tests/perf-test/OptionsListUtils.perf-test.ts

```diff
@@ -75,4 +82,50 @@ describe('OptionsListUtils', () => {
         await waitForBatchedUpdates();
         await measureFunction(() => OptionsListUtils.getMemberInviteOptions(personalDetails, mockedBetas, SEARCH_VALUE));
     });
-});
+
+    test('[OptionsListUtils] worst case scenario with a search term that matches a subset of selectedOptions, filteredRecentReports, and filteredPersonalDetails', async () => {
```

Comment:
Can you also add a scenario with an empty searchTerm so that[ this](https://github.com/Expensify/App/blob/a4d01a87b242eef06af37db965bd135745e342e5/src/libs/OptionsListUtils.ts#L1984) condition is met? Probably we will have to add Onyx mock for allPersonalDetails ( see [example](https://github.com/Expensify/App/blob/d5cf51d1e84e6647acb98c5d4545115b3bd08003/tests/perf-test/ReportUtils.perf-test.ts#L41))

Reply:
Are you sure this condition has been met? I think we need to set `shouldGetOptionDetails` to `true` for this to work

Reply:
Now it looks good 👍